src/status_im/ui/screens/communities/communities_list_redesign.cljs

```diff
@@ -202,9 +202,10 @@
                                           quo2.colors/neutral-50
                                           quo2.colors/neutral-40)}]]]))
 
-(defn views []
+(defn communities-list []
   (let [multiaccount (<sub [:multiaccount])
-        communities  (<sub [:communities/communities])]
+        communities (<sub [:communities/communities])
+        featured-communities (<sub [:communities/featured-communities])]
     (fn []
```

Comment:
I think Reagent generally recommends to use a form-1 component if there's no need for the form-2 variant. In this case, we don't need to close over any state.